### ISSUE TITLE
[Accessibility] The action buttons on the packages list is not accessible

### ISSUE THREAD

**Issue by anangaur** [NuGet/NuGetGallery#6403]
We should ensure all page elements can be navigated using a screen reader. Eg: 
![image](https://user-images.githubusercontent.com/14800916/45246692-6d75dc00-b2b7-11e8-8cf7-e4f5ae5e0a20.png)

**Comment by ryuyu** [comment]
I was unable to reproduce this issue, even using the application that was in the original video.
